ui/components/app/edit-gas-fee-popover/network-status/status-slider/status-slider.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+const GradientColors = [
+  '#037DD6',
+  '#1876C8',
+  '#2D70BA',
+  '#4369AB',
+  '#57629E',
+  '#6A5D92',
+  '#805683',
+  '#9A4D71',
+  '#B44561',
+  '#C54055',
+];
+
+const StatusSlider = () => {
+  const statusValue = 0.5;
+  const sliderValueNumeric = Math.round(statusValue * 10);
+  return (
+    <div className="status-slider">
+      <div
+        className="status-slider__arrow"
+        style={{
+          borderTopColor: GradientColors[sliderValueNumeric],
+          marginLeft: `${sliderValueNumeric * 10 - 10}%`,
+        }}
+      />
+      <div className="status-slider__line" />
+      <div
+        className="status-slider__label"
+        style={{ color: GradientColors[sliderValueNumeric] }}
+      >
+        {statusValue <= 0.5 ? 'Stable' : 'Unstable'}
```

Comment:
Maybe this isn't a big deal now as we have to come back to this anyway, but didn't this end up getting changed to three levels? I'm seeing "Not busy", "Stable", and "Busy".

Reply:
Good point, I need to check this with product team.